public/js/components/stories/utils.js

```diff
@@ -5,12 +5,12 @@ const { combineReducers } = require("redux");
 const reducers = require("../../reducers");
 const { fromJS } = require("immutable");
 
-function createStore({ sources = {}, breakpoints = {}, pause = null } = {}) {
-  return configureStore({})(combineReducers(reducers), {
-    sources: fromJS({ sources }),
-    breakpoints: fromJS({ breakpoints }),
-    pause: fromJS({ pause })
-  });
+function createStore(state = {}) {
+  for(var k in state) {
+    state[k] = fromJS(state[k]);
+  }
```

Comment:
Yeah. `mapValues` might be something we use enough to have as a utility lib without pulling down all of lodash. If we end up being able to use webpack for real in the long-term, I'd be fine requiring specific lodash functions in.